front/src/modules/trainschedule/components/ManageTrainSchedule/TrainSettings.tsx

```diff
@@ -91,8 +94,8 @@ export default function TrainSettings() {
           id="trainSchedule-name"
           onChange={(e: React.ChangeEvent<HTMLInputElement>) => setName(e.target.value)}
           value={name}
-          isInvalid={!name}
-          errorMsg={t('errorMessages.mandatoryField')}
+          isInvalid={invalidName}
+          errorMsg={`${t('errorMessages.requiredField')} ${t('errorMessages.nameLengthLimit')}`}
```

Comment:
Ah actually, I thought that you could differentiate the 2 cases:
- if the name is not defined, then we should display the requiredField
- if the name is defined but too long, we should display the nameLengthLimit message

Reply:
Done